projects/EfficientDet/README.md

```diff
@@ -22,45 +22,29 @@ In contrast to other feature pyramid network, such as FPN, FPN + PAN, NAS-FPN, B
 
 ## Usage
 
-### Model conversion
```

Comment:
The conversion guide can still be kept to provide a way for users to inference the originally trained model. We can have different settings in the training re-implementation and we should discuss the difference in this readme